substrate/frame/staking/src/pallet/mod.rs

```diff
@@ -2146,6 +2180,19 @@ pub mod pallet {
 			);
 			Ok(())
 		}
+
+		#[pallet::call_index(30)]
```

Comment:
how about a doc? (=

Reply:
Ledger update does not migrate. I did it this way since 1. its safe (just extra locked funds), and 2. no extra read/write while ledger updates since this code can linger for a while. But practically, we will migrate all accounts via a bot, so there will be no extra locked funds.